_sections/live-share-code-with-me-support-vscode.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Visual Studio Code
+tab: vscode
+---
+
+#### Starting a Live Share Session 
+If you have the Live Share extension installed on VS Code, you have the option of using Live Share to pair program on your call. To start a Live Share session, you should select the Live Share tab on the Share Code popup modal, and select the project you would like to share. 
+
+![Start Live Share](/uploads/codeshare-start-liveshare.jpg "Start Live Share"){:class="screenshot"}
+
+Once you have selected a project you will be prompted to switch back to your IDE.
+
+![Live Share Open Repo](/uploads/liveshare-open-repo.jpeg "Live Share Open Repo"){:class="screenshot"}
+
+If you have already installed Live Share and you haven't signed in yet, you will be prompted to sign in. 
```

Comment:
hm might have to mention installation but depends on your answer to my question of the installation live share